include_core/AtomicSupport.hpp

```diff
@@ -369,13 +374,7 @@ class VM_AtomicSupport
 #endif /* defined(__riscv) */
 #elif defined(_MSC_VER) /* defined(__GNUC__) */
 		return (uint32_t)_InterlockedCompareExchange((volatile long *)address, (long)newValue, (long)oldValue);
-#elif defined(J9ZOS390) /* defined(_MSC_VER) */
-		/* V1.R13 has a compiler bug and if you pass a constant as oldValue it will cause c-stack corruption */
-		volatile uint32_t old = oldValue;
-		/* 390 cs() function defined in <stdlib.h>, doesn't expand properly to __cs1() which correctly deals with aliasing */
-		__cs1((uint32_t *)&old, (uint32_t *)address, (uint32_t *)&newValue);
-		return old;
-#else /* defined(J9ZOS390) */
+#else /* defined(_MSC_VER) */
 #error "lockCompareExchangeU32(): unsupported platform!"
 #endif /* defined(__xlC__) */
```

Comment:
as per the [coding standard](https://github.com/eclipse-omr/omr/blob/3223d8d0d584575eafb5ec462250baae92cd6084/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif), `#endif` should have a comment describing the matching ?`#if`
```suggestion
#endif /* defined(OMR_ARCH_POWER) && !defined(OMR_ENV_DATA64) */
```

Reply:
Looking at the code closer ... shouldn't this be 
```
 #endif /* defined(OMRZTPF) */
```

I am correcting as per I am seeing the order of the conditionals, please let me know per the newest upcoming commit if it looks okay.

Reply:
Shouldn't this be 
```
#endif /* defined(OMRZPTF) */
```

I will make the changes based on the conditional alignments I am seeing in the upcoming commit, let me know if it looks okay after that.

Reply:
yes, it should be `#endif /* defined(OMRZPTF) */`